pom.xml

```diff
@@ -22,7 +22,6 @@
         <module>modules/GraphvizLayout</module>
         <module>modules/GraphStreaming</module>
         <module>modules/DesktopStreaming</module>
-        <module>modules/StreamingAPI</module>
```

Comment:
Eduardo, 
As I modified the plugin, I did not know how to get the simple plugin, maybe I had to do git on the folder, (it was difficult also to find which was the branch  because it's not on master.) I'ts true that I could not compile the full project becasue the StreamingApi was giving me errors. and I removed it from Netbeans... I'll try to do a pull request of the single module
By the way I could not run Gephi from maven, I tried the module on a regular install of gephi.